src/fmu/dataio/dataio.py

```diff
@@ -48,12 +49,22 @@
 AggregatedData: Final = AggregatedData  # Backwards compatibility alias
 InitializeCase: Final = InitializeCase  # Backwards compatibility alias
 
-
 # ======================================================================================
 # Private functions
 # ======================================================================================
 
 
+def _future_warning_preprocessed() -> None:
+    warnings.warn(
+        "Using the ExportData class for re-exporting preprocessed data is no "
+        "longer supported. Use the dedicated ExportPreprocessedData class "
+        "instead. In a deprecation period the ExportPreprocessedData is used "
```

Comment:
Not the most critical comment... But is it necessary to state here that `ExportPreprocessedData` is used under the hood?

Reply:
probably not, but I was mainly saying it in the rare case they experienced a change in behaviour.. The `ExportPreprocessedData` class does not support updating filenames hence if users specify e.g. the argument `name` in `ExportData` it will not be populated into `ExportPreprocessedData`. 

I don't reckon anyone has done that though.. probably the users only have the ERT workflow `WF_COPY_PREPROCESSED_DATAIO`, and no argument is given in there to update the file name on the re-export.